wine-tkg-git/wine-tkg-patches/misc/fastsync/ntsync5-staging-protonify.patch

```diff
@@ -831,29 +831,29 @@ index edfeb03..0000000
 -
```

Comment:
Hi, thanks for this and your past contributions.

The next few lines here revert changes in #1293 and breaks builds on staging again.